www/components/Docs/SymbolLinkTable.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box } from "@welcome-ui/box";
+import { Link } from "@welcome-ui/link";
+import { Text } from "@welcome-ui/text";
+import NextLink from "next/link";
+import React from "react";
+import { UrlObject } from "url";
+
+import Table from "@/components/CondensedTable";
+
+import { sortedBy } from "@/lib/utils";
+
+import Documentation from "./Documentation";
+
+interface Sym {
+  name: string;
+  documentation: string[];
+}
+
+interface Props<T extends Sym> {
+  title: string;
+  symbols: T[];
+  url: (sym: T) => string | UrlObject;
+}
+
+function deduplicate<T extends Sym>(xs: T[]): T[] {
+  return sortedBy(xs, "name", "documentation")
+    .reverse()
+    .filter((x, i, xs) => {
+      return i === 0 || x.name !== xs[i - 1].name;
+    })
+    .reverse();
+}
+
+export default function SymbolLinkTable<T extends Sym>({
+  title,
+  symbols,
+  url,
+}: Props<T>) {
+  if (symbols.length === 0) {
+    return <></>;
+  }
+  return (
+    <Box>
+      <Text variant="h3">{title}</Text>
+      <Table>
+        <Table.Tbody>
+          {deduplicate(symbols).map((sym) => {
+            return (
+              <Table.Tr key={sym.name}>
+                <Table.Th>
+                  <NextLink href={url(sym)} passHref>
+                    <Link>{sym.name}</Link>
```

Comment:
We should strip the module prefix from this name - they don't provide much value but increase the size of the table